test/permit/permit2Nonce.test.ts

```
@@ -0,0 +1,105 @@
+    });
+    return BigInt(nonce);
+  } catch (error) {
+    console.error('Unable to get next permit2 nonce', { error });
+    throw error;
+  }
+};
+
+describe('Permit2 Nonce Tests', () => {
+  const account = '0x4ab9F97585B0161f1aDa8484B209C44be54dad73';
+
+  Object.entries(permitProxy).forEach(([chainIdStr, proxyAddress]) => {
```

ESLint

[error] 'proxyAddress' is defined but never used.

(@typescript-eslint/no-unused-vars)